engines/python/setup/djl_python/deepspeed.py

```diff
@@ -125,7 +125,7 @@ def initialize(self, properties: dict):
                 )
             kwargs = {
                 "max_batch_size":
-                int(properties.get("max_rolling_batch_size", 4)),
+                int(properties.get("max_rolling_batch_size", 32)),
```

Comment:
This might cause OOMs for existing use-cases since our deepspeed has a 2D KVCache implementation

Reply:
oh interesting, the default rb size on the java size was 32 before, does the python side for ds restrict it further if we set this lower?

Reply:
reverted this